docker-images/stunnel-zookeeper/Makefile

```diff
@@ -0,0 +1,5 @@
+PROJECT_NAME=stunnel-zookeeper
```

Comment:
As I said before, I think this should be named `zookeeper-stunnel`. Zookeeper is the primary thing here!